port/iotivity-liteConfig.cmake

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
This and iotivity-liteConfigVersion.cmake look like the files that are automatically generated by CMake. I don't think they should be commited.